### ISSUE TITLE
Linear accuracy is better than non linear 

### ISSUE THREAD

**Issue by ramdhan1989** [mwdhont/SimCLRv1-keras-tensorflow#8]
I am using the code for my own dataset but the accuracy of linear is much better than non linear. Also I have problem when running train_NL_and_evaluate with error printed is "Learning diverged, stopped." . 

please advise

thank you

**Comment by mwdhont** [comment]
Dear,

With such little information, it's hard to tell how I can help you. Note that the parameter choices for the NL training have been tuned to work well on the dataset at hand and that you might need to adapt them for your dataset.

Best,
Michiel
